docs/interviewing_options.md

```diff
@@ -1 +1,134 @@
 # Interviewing Options
+
+Overview the most common interviewing options to get a sense of how they are
+typically used, what is: involved, pros, cons, and biases.
+
+
+
+## Whiteboarding
+
+Whiteboarding can be the use a of a physical whiteboard or a online code/sharing
+editor where a candidate is given a problem and expected to come up with a
+solution within a specific timeframe, and normally think aloud their problem.
+
+Typically the problems candidates are given are algorithmic or puzzle type
+questions typically of what you mind find on [leetcode](https://leetcode.com) or [hackerrank](https://www.hackerrank.com/)
+
+#### Pros
+
+- identifying candidates whom are exceptionally brilliant, quick thinkers, and
+  extremely knowledgable about the specific technical topic at hand
+- See how well candidates can perform under pressure and scrutiny in real time.
+- If done online can help companies and candidates optimize on costs and time
+  of being on-site and in-person
+- Generally can be a low effort (cheap) way to screen out candidates whom might
+  not be technical enough for the role at hand. This can be effect in a company
+  where there are many many applicants.
+
+#### Cons
+
+- Can lead to a lot of false negatives
+- Can bias against people whom have family commitments (caregivers,
+  single parents, etc.)
+- Unintentionally discriminate based on age (favouring those whom might have
+  time to study, or have recently graduated from academia)
+- Creates a pressure cooker situation where candidates make mistakes[1]
+  - People whom might suffer from anxiety might perform well in these situations
+- Many of the questions asked in these formats are not real world type problems
+  and ultimately someone whom can invert a binary tree may not be able to
+  deliver on actual business problems faced by businesses
+- If candidate has potentially memorized the solution (from another interview
+  or seen it posted online) this again may not be indicative of their true
+  knowledge.
+- Unfair power imbalance between candidates and interviewers
```

Comment:
I am not sure I understand this one. 

Then I thought to myself -- would any of them be understood by everyone and how much should we explain further.

Reply:
Here I think I'm pointing out that in white boarding style situations people may just spend a lot of time memorizing type of questions, and the power imbalance is more to do with the fact the interviewer has all the answer, so it puts the interviewee in a awkward spot. Happy to remove these bit though. Maybe it doesn't make sense.